.docker/source/Dockerfile

```diff
@@ -4,12 +4,12 @@
 FROM moveit/moveit:master-ci-shadow-fixed
 MAINTAINER Dave Coleman dave@picknik.ai
 
+ENV ROS_UNDERLAY /root/ws_moveit/install
+WORKDIR $ROS_UNDERLAY/../src
```

Comment:
See point above about restoring `CATKIN_WS` as a variable. That way we can set `WORKDIR $CATKIN_WS/src` which I think is quite a bit cleaner